src/setup.c

```diff
@@ -49,19 +49,38 @@ static void setup_clock(void)
 }
 
 /**
- * @brief Initial interruptions configuration.
+ * @brief Exceptions configuration.
  *
+ * This function configures Nested Vectored Interrupt Controller for IRQ and
+ * System Control Block for system interruptions.
```

Comment:
Maybe you are missing a blank line after this paragraph.

Reply:
Ok